src/components/common/Extensions/ExtensionUpgradeButton/ExtensionUpgradeButton.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback } from 'react';
+
+import {
+  Extension,
+  ExtensionVersion,
+  isExtensionCompatible,
+} from '@colony/colony-js';
+import { ActionButton } from '~shared/Button';
+import { AnyExtensionData } from '~types';
+import { ActionTypes } from '~redux/index';
+import { mapPayload } from '~utils/actions';
+import { useAppContext, useColonyContext } from '~hooks';
+
+interface Props {
+  extensionData: AnyExtensionData;
```

Comment:
I think an extension can only be upgraded if it's installed, so could we make this `InstalledExtensionData` instead?